src/app/shared/form/button/button.component.ts

```diff
@@ -0,0 +1,49 @@
+import { NgModule, Component, OnInit, Input, ViewChild, ElementRef, Renderer2 } from '@angular/core';
+
+@Component({
+  selector: 'in-button',
+  templateUrl: './button.component.html',
+  styleUrls: ['./button.component.scss']
+})
+export class ButtonComponent implements OnInit {
+  @Input() fontSize: string;
```

Comment:
why such an overkill for styles? Isn't it better to expose one prop for class name or object for styles ?